components/auth-proxy-service/src/main/java/com/wso2telco/proxy/entity/Endpoints.java

```diff
@@ -791,5 +819,18 @@ private void createUserProfile(String username, String operator, String scope)
         userDTO.setUserName(username);
         userRegistrationAdminService.addUser(userDTO);
     }
+
+
+    private Map<String,String> validateAttributeShareScopes(String scopeName,String operatorName,String clientId,String loginhintMsis,String msisdn) throws AuthenticationFailedException{
```

Comment:
method is not required only a additional hop as a method

Reply:
Fixed